srcpkgs/buildifier/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'buildifier'
+pkgname=buildifier
+version=5.1.0
+revision=1
+wrksrc="buildtools-${version}"
+build_style=go
+go_import_path="github.com/bazelbuild/buildtools/buildifier"
+short_desc="Format bazel BUILD and .bzl files with a standard convention"
+maintainer="n1c00o <git.n1c00o@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/bazelbuild/buildtools/tree/master/buildifier"
+#changelog="https://github.com/bazelbuild/buildtools/releases"
```

Comment:
Either delete this comment or uncomment it and provide (preferably raw text) changelog.
```suggestion
```